omegaconf/dictconfig.py

```diff
@@ -682,3 +683,49 @@ def _dict_conf_eq(d1: "DictConfig", d2: "DictConfig") -> bool:
                 return False
 
         return True
+
+    def _to_object(self) -> Any:
+        """Instantiate an instance of `self._metadata.object_type`.
+        This requires `self` to be a structured config.
+        Nested subconfigs are converted to_container with resolve=True."""
+        from ._utils import get_structured_config_field_names
```

Comment:
You should be able to move this import at top of the file, no?

Also, minor formatting comment: most multiline docstrings in the code have `"""` on their own line.

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/b51d33f1135197cb1e204d2588b997f9e2b4f46c and https://github.com/omry/omegaconf/pull/502/commits/f1a4270bdae1c8f6fc219e8afe5a33c098892823.

Reply:
Thanks! The first `"""` is still on the same line but no biggie :)

Reply:
Fixed in https://github.com/omry/omegaconf/pull/502/commits/d12701e9add8ee7084b9ca8598870e441e974d6c :)